pkg/loadtest/client_kvstore.go

```diff
@@ -1,14 +1,29 @@
 package loadtest
 
-import "github.com/tendermint/tendermint/libs/common"
+import (
+	"encoding/binary"
+	"fmt"
+	"sync"
+
+	"github.com/tendermint/tendermint/libs/common"
+)
+
+const minKVStoreTxSize int = 40
```

Comment:
Not clear why 40 should be the minimum. Maybe add a comment explaining the rationale for this number, similar to comments inside function `ValidateConfig`.

Reply:
The broader question here is: what strategy do we use when constructing key/value pairs for the KVStore client?

Right now I'm trying to use as random an approach as possible to reduce chances of key collisions (to create unique transactions). Another approach would be a totally controlled one to ensure uniqueness, where each key is a combination of a client ID and a counter. In the latter case, the minimum transaction size must cater for a full load test with no duplicate transactions across all clients.

Unless one wants duplicate transactions?

Reply:
I think the current strategy is perfect for our purposes.

I can imagine a scenario where having duplicate transactions (accessing the same key) would be a useful feature, namely to test how contention affects performance. Other workload frameworks (e.g., YCSB) generate keys drawn from a specific distribution, such as uniform or zipfian, and offer thus the ability to create contention on the same key.

Such a feature, however, would be _useful for testing application-level performance primarily_, and would have no bearing on the consensus layer performance. For instance, high-contention workloads (zipfian) could test how efficient is the key-value store at handling high-contention access. Since we're currently only looking to investigate the consensus layer, maybe we can skip this feature and keep it in mind for later.

Reply:
`tm-load-test` allows you to build different clients with different kinds of behaviours, so it should be easy to eventually build an array of different clients to simulate different workloads.